src/modules/ModulesRouter.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Route, Switch } from 'react-router';
+
+import { Guard } from 'core/auth';
+
+import { withLazy } from 'utils';
+
+const Home = withLazy(() => import('./home'));
+const Login = withLazy(() => import('./login'));
+const Main = withLazy(() => import('./main'));
+const Register = withLazy(() => import('./register'));
+
+const ModulesRouter = () => {
+  return (
+    <Switch>
+      <Guard.UnprotectedRoute exact path="/login" redirect="/app" component={Login} />
+      <Guard.UnprotectedRoute exact path="/register" redirect="/app" component={Register} />
+      <Route path="/app" component={Main} />
+      <Route exact path="/" component={Home} />
+      <Route path="**" render={() => <div>Not Found Page </div>} />
+    </Switch>
+  );
```

Comment:
Shouldn't we declare other routes as unprotected aswell? Just to keep everything consistent

Reply:
Unprotected refers to - redirect somewhere if user is authorized - so authorized user cannot use this resources.

Protected refers to - redirect user to somewhere if he is not authorized - so he can access this resources only when he is auth,

Normal Route refers to - i dont care both authorized and not authorized users can access this resources - so this is ok

Reply:
Ah ok, sure. Got it.